api/publish.go

```diff
@@ -585,18 +587,22 @@ func apiPublishSourcesCreate(c *gin.Context) {
 	})
 }
 
-// @Summary Get staged source list
-// @Description **Get the staged source list**
+// @Summary List pending changes
+// @Description **List changes to be applied**
+// @Description
+// @Description Returns added, removed or changed components of snapshots or local repository to be published.
+// @Description
+// @Description The changes will be applied by a subsequent publish update call (See `PUT /api/publish/{prefix}/{distribution}`).
 // @Tags Publish
 // @Param prefix path string true "publishing prefix"
 // @Param distribution path string true "distribution name"
 // @Produce json
-// @Success 200 {array} sourceParams
+// @Success 200 {array} []deb.PublishedRepoRevision
```

Comment:
This not correct. `GET` just returns a list of sources. The revision is only used internally.